### PR TITLE
May 17 Exec

### PR DESCRIPTION
**Pull request by astronautthis**
@LongForWisdom, items to be clarified marked as (--> $ <--)

### CODE REVIEW

File: governance/votes/Executive Vote - May 17, 2023.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+### DAI Budget Streams
+
+As per [MIP107](https://mips.makerdao.com/mips/details/MIP107), the following DAI budget streams will be created if this executive proposal passes:
+
+#### DAI Budget Streams
+
+| Budget | DAI Amount | Start Date | End Date | Destination Address |
+|---|---|---|---|---|
+| Governance Security Engineering | 2,200,000 | 2023-05-01 | 2024-05-01 (--> $check date since it's midnight <---) | [0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0](https://etherscan.io/address/0x569fAD613887ddd8c1815b56A00005BCA7FDa9C0) (--> $source of truth for this address other than spreadsheet? <---) |
```
Comment (LongForWisdom) [inline governance/votes/Executive Vote - May 17, 2023.md:69]: Source of truth is the spreadsheet. Retro should be posting something you can link to as well.

File: governance/votes/Executive Vote - May 17, 2023.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ Multichain Engineering | 2,300,000 | 2023-05-01 | 2024-05-01 (--> $check date since it's midnight <---) | [0x868B44e8191A2574334deB8E7efA38910df941FA](https://etherscan.io/address/0x868B44e8191A2574334deB8E7efA38910df941FA) (--> $source of truth for this address other than spreadsheet? <---)|
+
+Note that these streams are implemented as a [dssvest stream](https://manual.makerdao.com/module-index/module-token-streaming) that is controlled by the Protocol Engineering Facilitators, and paid out to Ecosystem Actors in order to do work necessary for Governance Security Engineering and Multichain Engineering to function properly.
+
+### Data Insights Core Unit (DIN-001) MKR Transfer
+
+As per their successful [MKR vesting proposal](https://mips.makerdao.com/mips/details/MIP40c3SP64#mkr-vesting) **103.16 MKR** (--> $source of truth for MKR amount apart from spreadsheet? <--)will be transferred to the [Data Insights Core Unit (DIN-001)](https://mips.makerdao.com/mips/details/MIP39c2SP22) wallet [0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F](https://etherscan.io/address/0x7327Aed0Ddf75391098e8753512D8aEc8D740a1F), if this executive proposal passes.
```
Comment (LongForWisdom) [inline governance/votes/Executive Vote - May 17, 2023.md:76]: Spreadsheet is source of truth, Tadeo confirmed it, and its less than stated in the MIP.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments. Thanks Astro

File: governance/votes/Executive Vote - May 17, 2023.md (hunk 1 above)
Comment (astronautthis) [reply governance/votes/Executive Vote - May 17, 2023.md:69]: I don't see a post yet so link to be added.

**Comment by astronautthis** [comment]
Almost all done, just waiting on Retro's post for a link to verify the addresses of the Dai budget streams

**Pull request merged**